common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/util/classpath/ClasspathUtil.xtend

```diff
@@ -108,16 +108,19 @@ class ClasspathUtil {
 		}
 	}
 
-	def protected IPath getEclipseClasspathEntry(IPath path) {
+	def IPath getEclipseClasspathEntry(IPath path) {
+		logger.info("Get classpath entries from workspaceHelper using path='{}'", path)
 		val classpathEntries = getSourceClasspathEntries(workspaceHelper.root.getFile(path).project)
+		logger.debug("Found eclipse classpath entries = '{}'", classpathEntries.join(', '))
 		return classpathEntries.findFirst[it.path.isPrefixOf(path)]?.path
 	}
 
 	def Iterable<IClasspathEntry> getSourceClasspathEntries(IProject project) {
-		if (project.hasJavaNature) {
+		try {
 			val javaProject = JavaCore.create(project)
-			return javaProject.rawClasspath.filter[entryKind == IClasspathEntry.CPE_SOURCE]
-		} else {
+			return javaProject?.rawClasspath.filter[entryKind == IClasspathEntry.CPE_SOURCE]
```

Comment:
In case `javaProject` is null this will still fail. You would need to write `javaProject?.rawClasspath?.`